app/views/carts/show.html.erb

```diff
@@ -4,33 +4,33 @@
 <table>
   <thead>
     <tr>
-        <th data-field="image">Image</th>
-        <th data-field="item-name">Item</th>
-        <th data-field="quantity">Quantity</th>
-        <th data-field="change-quantity">Change Quantity</th>
-        <th data-field="price">Price</th>
-        <th data-field="price">Subtotal</th>
-        <th data-field="remove">Remove</th>
+        <th data-field="image"><h5>Image</h5></th>
+        <th data-field="item-name"><h5>Item</h5></th>
+        <th data-field="quantity"><h5>Quantity</h5></th>
+        <th data-field="change-quantity"><h5>Change Quantity</h5></th>
+        <th data-field="price"><h5>Price</h5></th>
+        <th data-field="price"><h5>Subtotal</h5></th>
+        <th data-field="remove"><h5>Remove</h5></th>
     </tr>
   </thead>
 
   <tbody>
     <% @cart.contents.each do |item, quantity| %>
     <tr>
       <td><img src="<%=Item.find(item).image_tag%>"/></td>
```

Comment:
Do we want to take this AR logic out of this view somehow? @meyerhoferc @CjMoore @valerietrudell